internal/core/api.rs

```diff
@@ -571,6 +596,14 @@ impl Window {
     pub fn is_visible(&self) -> bool {
         self.0.is_visible()
     }
+
+    /// Returns a struct that implements the raw window handle traits to access the windowing system specific window
+    /// and display handles.
+    #[cfg(feature = "raw-window-handle-06")]
+
```

Comment:
extra line.

The docs could add link to the traits being implemented.
Should the doc also mention the feature?